src/hera/task.py

```diff
@@ -6,6 +6,7 @@
 import textwrap
 from typing import Any, Callable, Dict, List, Optional, Set, Union
 
+from argo_workflows.model.security_context import SecurityContext
```

Comment:
What do you think about importing this in the block starting on line 16?

Reply:
I have put it there because `model` comes before `model_utils` and `models`, but I can change it, whatever you choose.

Reply:
The other comments I have fixed, please update me If you'd like me to change this one as well.